source/administration/changelog.md

```diff
@@ -145,11 +145,13 @@ The following deprecations are planned for the Mattermost v6.0 release, which is
 
 5. Windows 7 reached [EOL in January 2020](https://support.microsoft.com/en-us/windows/windows-7-support-ended-on-january-14-2020-b75d4580-2cc7-895a-2c9c-1466d9a53962). We will no longer provide support for the Desktop App issues on Windows 7.
 
-6. [DisablePostMetaData](https://docs.mattermost.com/administration/config-settings.html#disable-post-metadata) configuration setting.
+6. [DisableLegacyMFAEndpoint](https://docs.mattermost.com/administration/config-settings.html#disable-legacy-mfa-api-endpoint) configuration setting.
```

Comment:
I wouldn't normally do it this way, but since we have this PR open, I just took this opportunity to add a new deprecated setting that was just communicated to me. https://community.mattermost.com/core/pl/198pxrsksfbwjxk89t1ggi4qgo